src/lib/style-properties.html

```diff
@@ -285,6 +285,13 @@
         return {properties: props, key: o};
       },
 
+      _rootSelector: /(?:^:root)|(?::host\s*>\s*\*)/,
```

Comment:
It is being tested on the parsedSelector, so `:root` will probably be there because we don't replace it before the parsing.